package/CHANGELOG

```diff
@@ -13,11 +13,14 @@ The rules for this file:
   * release numbers follow "Semantic Versioning" http://semver.org
 
 ------------------------------------------------------------------------------
-mm/dd/17 richardjgowers, rathann, orbeckst
+mm/dd/17 richardjgowers, rathann, orbeckst, kain88-de
 
   * 0.17.0
 
 Enhancements
+  * libmdaxdr classes now accept more argument types for a write (PR #1442)
+  * libmdaxdr classes now raise EOFError when accessing another frame after
```

Comment:
I think this counts as a "Change", not an enhancement. State what it used to do, IOError,